__tests__/consumer.integration.test.js

```diff
@@ -74,6 +74,18 @@ describe('Tests while consumer is connected', () => {
     expect(Array.isArray(meta.topics)).toBe(true);
   });
 
+  test('Should return topic offsets', async () => {
+    const response = await kafkaConsumer.getTopicOffsets(testTopic, 0, 5000);
+
+    expect(response).toEqual({ highOffset: 0, lowOffset: 0 });
+  });
+
+  test('Should reject with no args passed in.', async () => {
+    await kafkaConsumer.disconnect();
+
+    await expect(kafkaConsumer.getTopicOffsets(testTopic, 0, 5000)).rejects.toBeDefined();
```

Comment:
The description of this test is incorrect. Since this failure is actually caused by the fact that the consumer is disconnected.